### ISSUE TITLE
[Theory] What would be an acceptable average loss at the end of training?

### ISSUE THREAD

**Issue by FilipaRamos** [AlexeyAB/darknet#419]
Hi @AlexeyAB !

I would like to know what would be a normal average loss value. And is there anything specific I can do in order to drop my average loss even further? 

Thanks!

**Comment by AlexeyAB** [comment]
@FilipaRamos Hi, there isn't specific avg_loss, it depends on training dataset and on the model. I saw final avg_loss from 0.03 to 3.0

To drop training avg_loss you can:
* If you previously trained without a pre-trained model, then train with pre-trained weights that corresponds to your model (cfg-file) - `darknet19_448.conv.23` for yolo-voc.2.0.cfg and yolo.2.0.cfg, and for other models: https://github.com/AlexeyAB/darknet#how-to-train-tiny-yolo-to-detect-your-custom-objects
* you can try to generate your own anchors: https://github.com/AlexeyAB/darknet/issues/418#issuecomment-369890346
* also try to increase network resolution in your cfg-file, for example, `width=832 height=832` to the same value multiple of 32

**Comment by FilipaRamos** [comment]
Sorry for the late answer @AlexeyAB, I think my loss is good then, it's around 0.5. Thanks a lot!
